pages/api/capture-event.js

```diff
@@ -28,7 +28,7 @@ export default withContainer(
       return;
     }
 
-    if (!body.visitId || isNaN(body.visitId)) {
+    if (body.visitId === null || body.visitId === "" || isNaN(body.visitId)) {
```

Comment:
Could you clarify why we would want to accept 0 as a visitId ? (based on the message in a8ce8e9d7439c).

If we want to explicitly allow 0, have we considered including that in the statement?

(e.g. `if ((!body.visitId && body.visitId !== 0) || isNaN(body.visitId))`

This should also cover an undefined visitId

Reply:
I think at the time we reckoned 0 was a valid `visitId`. Now I'm wondering whether it is, looking at the DB?

Reply:
The default minimum value is 1 so we should never had visits with an ID of 0 (https://www.postgresql.org/docs/6.4/sql-createsequence.htm)

Reply:
Awesome, I'll drop the commit :)